plugin-catalog/src/components/plugins/InstalledList.tsx

```diff
@@ -95,5 +180,11 @@ export function PluginInstalledList() {
     fetchInstalledPlugins();
   }, []);
 
-  return <PurePluginInstalledList installedPlugins={installedPlugins} error={error} />;
+  return (
+    <PurePluginInstalledList
+      installedPlugins={installedPlugins}
+      otherInstalledPlugins={otherInstalledPlugins}
```

Comment:
from the prop names it's not clear what's the difference between installedPlugins and otherInstalledPlugins